host/src/pages/Dashboard/Shortlet/shortletDetails.tsx

```diff
@@ -11,7 +41,82 @@ export default function ShortletDetails() {
         </BreadcrumbItem>
         <BreadcrumbItem>Shortlet name</BreadcrumbItem>
       </Breadcrumbs>
-      <div>shortletDetails</div>
+      <div className="space-y-4">
+        <div className="space-y-2">
+          <h2 className="text-shorttiee_primary text-lg font-bold">Apartment Name</h2>
+          <div className="flex gap-4">
+            <p className="text-shorttiee_primary font-semibold">Duplex</p>
+            <p className="text-sm">
+              <span className="text-base text-shorttiee_green-dark font-semibold">$40k</span> /night
+            </p>
+            <Chip
+              radius="sm"
+              size="sm"
+              className="text-shorttiee_yellow-dark bg-shorttiee_yellow-light text-xs font-normal capitalize"
+            >
+              pending
+            </Chip>
+          </div>
+        </div>
+        <div className="w-full  px-12">
+          <Carousel
+            opts={{
+              align: "start",
+            }}
+            className="w-full"
+          >
+            <CarouselContent>
+              {Array.from({ length: 5 }).map((_, index) => (
+                <CarouselItem key={index} className="md:basis-1/3">
+                  <div className="p-1">
+                    <img
```

Comment:
Use [nextUi image](https://nextui.org/docs/components/image) component so we could get fallbacks